spk/sickchill/src/service-setup.sh

```diff
@@ -62,10 +55,12 @@ service_postupgrade() {
 
 service_preupgrade ()
 {
-    # We have to reset /env folder to 3.8 so remove entire folder as it gets rebuilt in postinst and this avoids any conflicts.
-    # Revision 1 was python 3.7. For cleaner update remove share and lib folders for clean install, leave user data /var.
+    # We have to reset /env for Python and package changes, it gets rebuilt in postinst and this avoids any conflicts.
+    # For cleaner update remove bin, env, share and lib folders for fresh install, leave user data /var & /@appdata
```

Comment:
I'm surprised by this as normally the entire `target/*` directory (e.g. `${SYNOPKG_PKGDEST}) gets deleted at upgrade time.  Does this really have an effect?